sui_mamoru_relay/sources/incidents.move

```diff
@@ -0,0 +1,458 @@
+module sui_mamoru_relay::incidents {
+    use sui::object::{Self, UID};
+    use sui::linked_table::{Self, LinkedTable};
```

Comment:
linked table is like list, it will require to iterate over all items...

Reply:
Iterate all items for what? Check the usage code in `get_incidents_since`.. It seems like the linked table suits well here.